Content.Server/GameObjects/Components/Fluids/PuddleComponent.cs

```diff
@@ -106,8 +104,7 @@ public override void Initialize()
             _spriteComponent = Owner.GetComponent<SpriteComponent>();
             var robustRandom = IoCManager.Resolve<IRobustRandom>();
             var randomVariant = robustRandom.Next(0, _spriteVariants - 1);
-            // TODO: This is ugly
-            string[] splitRSI = _spriteComponent.BaseRSIPath.Split("/".ToCharArray());
+            string[] splitRSI = _spriteComponent.BaseRSIPath.Split(ResourcePath.SYSTEM_SEPARATOR);
```

Comment:
I meant that more as in `var baseName = new ResourcePath(_spriteComponent.BaseRSIPath).FileName.Replace("rsi", "");`